README.md

```diff
@@ -22,7 +25,17 @@ sbt "project codegalaxy-app" fastOptJS
 Then start the application locally, in simulator:
 ```bash
 cd app
+npm install
```

Comment:
I would move this two lines to Re-install (first setup) section, since they not needed for each start of app.

Reply:
ok